sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-26.mdx

```diff
@@ -42,7 +42,16 @@ Nicolas will make updates to the guidelines that all assertions should go into a
 
 Júlia is coordinating an upcoming bulk update of all modules in https://github.com/nf-core/modules/issues/5828 (join #wg-modules-update to stay tuned)
```

Comment:
when we mention names, can we link to their github account. makes it feel less insidery

Reply:
dione